homeassistant/components/zha/light.py

```diff
@@ -380,6 +383,12 @@ async def async_added_to_hass(self):
         self._cancel_refresh_handle = async_track_time_interval(
             self.hass, self._refresh, timedelta(seconds=refresh_interval)
         )
+        await self.async_accept_signal(
```

Comment:
For a differenet PR, should we change `self.async_accept_signal` from `async` to be `@callback` ?

Reply:
https://github.com/home-assistant/core/pull/38009